examples/pytorch-fashion-mnist/pytorch-fashion-mnist.ipynb

```diff
@@ -368,35 +382,18 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# \"pip install\" a BentoML archive\n",
-    "\n",
-    "BentoML user can directly pip install saved BentoML archive with `pip install $SAVED_PATH`,  and use it as a regular python package."
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "!pip install ./model"
+    "### Run a BentoML service"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "scrolled": true
+   },
    "outputs": [],
    "source": [
-    "# Your bentoML model class name will become packaged name\n",
-    "import FashionMNISTModel\n",
-    "\n",
-    "ms = FashionMNISTModel.load()\n",
-    "result = ms.predict(testing_data)\n",
-    "log_probs, output_classes = result.max(dim=1)\n",
-    "\n",
-    "accuracy = accuracy_score(targets.numpy(), output_classes.data.numpy())\n",
-    "print('Accuracy: {:.2%}'.format(accuracy))"
+    "!bentoml serve ./model"
```

Comment:
jupyter notebook allows using python variable in this type of bash command: `!bentoml serve {saved_path}`

the previous cell of moving from saved_path to './model' can be deleted as well

Reply:
Ok, will change.